internal/pkg/admission/policy-server-deployment.go

```diff
@@ -226,6 +226,19 @@ func (r *Reconciler) adaptDeploymentSettingsForPolicyServer(policyServerDeployme
 			},
 		)
 	}
+	if r.MetricsEnabled {
+		policyServerDeployment.Annotations[constants.OptelInjectAnnotation] = "true"
+	}
+}
+
+func envVarsContaintVariable(container *corev1.Container, envVarName string) (bool, int) {
+	for i, envvar := range container.Env {
+		if envvar.Name == envVarName {
+			return true, i
+		}
+	}
+	return false, -1
+
```

Comment:
nitpicking...

* The name of the function has a typo. I think you meant to call it `envVarsContainsVariable`
* I don't like the return type of the function, I see two options:
  *  return `int`: it would be `-1` if the item has not been found, otherwise it would return the index position
  * return `(bool, string)`: instead of returning the index to the value, just return the value

I also wonder if it would make sense to make the function more generic by changing the signature to use that as input: `(values []EnvVar, key string)`

Reply:
You're right, `contain` is correct. However the name currently is "envVarsContain**t**Variable", note the highlighted "t". It should be `envVarsContainVariable`